HACKING.md

```diff
@@ -18,10 +18,8 @@ dnf install -y oci-kvm-hook # if on traditional
 ```
 
 We will use `oc cluster up` to set up a local cluster for testing. To do
-this, simply obtain the OpenShift binary from
-[the latest release](https://github.com/openshift/origin/releases/latest)
-(though note that I'm currently still using the binaries from v3.9, so
-pick that release if you hit issues).
+this, simply obtain the OpenShift v3.6.1 binary from
+[here](https://github.com/openshift/origin/releases/tag/v3.6.1).
```

Comment:
maybe we should mention why we're choosing 3.6.1 (or do we do that somewhere else).

Reply:
We do mention it down, but this is the first time, so I moved the explanation here instead.